spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/convert/StructAccessor.java

```diff
@@ -170,4 +174,13 @@ private Class getSingleItemTypeCode(Type colType) {
 				? SpannerTypeMapper.getArrayJavaClassFor(colType.getArrayElementType().getCode())
 				: SpannerTypeMapper.getSimpleJavaClassFor(code);
 	}
+
+	public <T> T getSingleJsonValue(String colName, Class<T> colType) {
+		if (this.struct.isNull(colName)) {
+			return null;
+		}
+		String jsonString = this.struct.getJson(colName);
+		Gson gson = new Gson();
```

Comment:
I don't think you need to create a `Gson` object on every call. You can make it a member variable. It's supposed to be thread-safe.

Reply:
Good point. I'll change it.